index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
+deliverables of the Working Group will be published and maintained:
+          </p>
+          <table>
+            <tr>
+              <th>Registry</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
```

Comment:
"Verifiable Credentials Extensions Registry" is far too generic.  There almost certainly will be multiple kinds of extensions, each of which should have its own registry.

As an illustrative example, the specification https://www.rfc-editor.org/rfc/rfc8809.html creates registries for two different specific types of WebAuthn extensions.  Similarly, the set of registries at https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml are for eleven distinct kinds of OAuth extensions.

Reply:
Fixed in fb17a5abed3d92b62ed4d4d918d81a80426e56de.